submissions/kikinovk/JS_DOM_API/js/app.js

```diff
@@ -0,0 +1,93 @@
+/*
+* 
+*/
+const list  = [
```

Comment:
The name of this variable is too generic. Let's add more concretics to this variable, please